src/main/java/com/twilio/rest/api/v2010/account/usage/record/AllTimeReader.java

```diff
@@ -201,16 +172,12 @@ private void addQueryParams(final Request request) {
             request.addQueryParam("Category", category.toString());
         }
         
-        if (absoluteStartDate != null) {
-            request.addQueryParam("StartDate", absoluteStartDate.toString(Request.QUERY_STRING_DATE_FORMAT));
-        } else if (rangeStartDate != null) {
-            request.addQueryDateRange("StartDate", rangeStartDate);
+        if (startDate != null) {
+            request.addQueryParam("StartDate", DateConverter.dateStringFromLocalDate(startDate));
```

Comment:
consider adding an overloaded method on Request `addQueryParam(LocalDate)` which does the conversion to a string in one place rather than having to put `DateConverter.dateStringFromLocalDate()` everywhere?